sale_order_lot_selection/__manifest__.py

```diff
@@ -1,3 +1,6 @@
+# Copyright (C) 2025 Cetmix OÜ
```

Comment:
You should not put your self here as the original author is AgileBg.

See: https://github.com/OCA/sale-workflow/commit/985a25edf0cf39fca7355f20c048ebdb6a14ff8e#diff-353971a56f991d9f92bb53b56542dd48451663ed145117879757d7f2e7eac9eaR4

You should instead restore the original authorship in files (it seems they have been lost)